reports/report_details.py

```diff
@@ -146,7 +146,7 @@ def add_transnational_to_regions(regions):
         '2015': {
             'name': '7',
             'title': 'Women in the news (sources) by medium',
-            'desc': 'Breakdown by sex of all mediums',
+            'desc': 'Breakdown by sex of all 3 mediums (ADD INTERNET AND TWITTER)',
```

Comment:
```suggestion
            'desc': 'Breakdown by sex of all mediums',
```

1. I think  (ADD INTERNET AND TWITTER) was a directive and not part of the title.
2. There is also a suggested change in terms of which Model to use [here](https://docs.google.com/spreadsheets/d/1FV6pUcW3hEade4y17ENGAP5Dp4RkNnDW/edit#gid=1308543941&range=C3). Have you looked at it?